api/graphql/query/event.js

```diff
@@ -1,8 +1,17 @@
 import Models from 'Models';
 import BaseEntityResolver from 'Graphql/base/EntityResolver';
+import { UserInputError } from 'apollo-server';
 
 class Event extends BaseEntityResolver {
   model = Models.CalendarEvent;
+
+  validate = () => {
```

Comment:
It is mentioned in the issue to update the BaseEntityResolver to handle this based on if the provided model has the slug attribute.